src/test/java/parser/JsonParserTest.java

```diff
@@ -0,0 +1,64 @@
+package parser;
+import com.google.gson.Gson;
+import org.junit.jupiter.api.*;
+import shop.Cart;
+import shop.RealItem;
+
+import java.io.*;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class JsonParserTest {
+
+    RealItem realItem;
+    Parser parser = new parser.JsonParser();
+    Cart userCart;
+
+    @BeforeEach
+    void setUp() {
+        userCart = new Cart("user-cart");
+        realItem = new RealItem();
+        realItem.setName("Book");
+        realItem.setWeight(400);
+        realItem.setPrice(21.50);
+        userCart.addRealItem(realItem);
+
+        parser.writeToFile(userCart);
+    }
+
+    @Test
+    @DisplayName("New File can be successfully created")
+    void writeToFile() {
+        parser.writeToFile(userCart);
+        File expectedFile = new File("src/main/resources/user-cart.json");
+        assertTrue(expectedFile.exists());
```

Comment:
The correct verification will be to read the file using Gson and compare that fields are the same